src/odemis/util/conversion.py

```diff
@@ -164,6 +165,41 @@ def hex_to_frgba(hex_str, af=1.0):
     """
     return rgba_to_frgba(hex_to_rgba(hex_str, int(af * 255)))
 
+# from claude / ome_types
+def rgba_to_int32(rgba: Tuple[int]) -> int:
+    """Convert an RGBA tuple to a signed 32-bit integer representation.
+    :param rgb: A tuple of values (Red, Green, Blue, Alpha)
```

Comment:
Param is now called `rgba`. Also, please mention the facts that the values should be 0-255.